source/renderer/app/themes/daedalus/cardano.ts

```diff
@@ -871,7 +871,7 @@ export const CARDANO_THEME_OUTPUT = {
     '--theme-staking-stake-pool-tooltip-table-title-color': '#5e6066',
     '--theme-staking-stake-pool-tooltip-text-color': '#5e6066',
     '--theme-staking-stake-pools-search-button-color': '#5e6066',
-    '--theme-staking-stake-pools-search-icon-color': 'rgba(94, 96, 102, 1)',
+    '--theme-staking-stake-pools-search-icon-color': '#5e6066',
```

Comment:
rgba(94, 96, 102, 1) = #5e6066 
this fixes nothing ))

Reply:
This is a way to standardize with the other theme variables.

Reply:
well... why only cardano theme then?

Reply:
Will also do it for light blue. The others seem to be fine.